packages/docs/content/foundation/typography/headings.mdx

```diff
@@ -131,7 +131,7 @@ Heading--4xl is used for the heading on landing pages that serve as an entry poi
 
 
 <ResponsiveExample
-  storyId="foundations-typography--responsive-headings"
+  storyId="foundations-typography-headings--responsive-headings"
```

Comment:
I'm not seeing Storybook examples embedded here.

Reply:
Hmmm, I'm not able to reproduce. This is from me running locally 🤷 
![Screen Shot 2022-10-28 at 4 58 45 PM](https://user-images.githubusercontent.com/1449852/198731275-f6009690-9029-43dd-8c2c-d8bbdd6adf18.png)

Reply:
@zarahzachz, could you have forgotten to `yarn build:storybook:docs`?

Reply:
Yeah, that's exactly what happened. Disregard my feedback!